has-value.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'has-value' {
+  interface Options {
+    isValid: (key: string, obj: object) => boolean;
```

Comment:
Codacy found an issue: [Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys.](https://app.codacy.com/gh/spacebin-org/spirit/pullRequest?prid=6133005)